plasmapy/diagnostics/tests/test_charged_particle_radiography.py

```diff
@@ -612,7 +611,7 @@ def test_saving_output(tmp_path):
     results_1 = sim.results_dict
 
     # Save result
-    path = os.path.join(tmp_path, "temp.npz")
+    path = str(tmp_path.joinpath("temp.npz"))
```

Comment:
I always forget if it is, but if tmp path is the pathlib.Path variant, you can do


```suggestion
    path = str(tmp_path / "temp.npz")
```
Which calls joinpath under the hood!